File: Domain/ProjectTrackingTool.cs
using System.Collections.Generic;
using ReleaesNotesGenerator.Common.Enums;

namespace ReleaseNotesGenerator.Domain
{
    public class ProjectTrackingTool : EntityBase<int>
    {
        public string Name { get; set; }

        public string Url { get; set; }        

        public string AccessToken { get; set; }

        public string ProjectName { get; set; }

        public ProjectTrackingToolType Type { get; set; }

        public List<Repository> Repositories { get; set; }
    }
}

File: ReleaseNotesGenerator/ClientApp/app/common/interfaces/ProjectTrackingTool.ts

    import { Repository } from './Repository';
import { ProjectTrackingToolType } from './ProjectTrackingToolType';
import { EntityBase } from './EntityBase';
    export class ProjectTrackingTool extends EntityBase<number> {
        
        name: string
        url: string
        projectName: string
        type: ProjectTrackingToolType
        repositories: Repository[]
        }

    